test/hotspot/jtreg/runtime/cds/appcds/AOTFlags.java

```diff
@@ -25,13 +25,14 @@
 /*
  * @test
  * @summary "AOT" aliases for traditional CDS command-line options
- * @requires vm.cds
+ * @requires vm.cds & vm.compMode != "Xcomp"
```

Comment:
The test completely ignore external VM flags, so it should have  
`@requires vm.flagless `
and no Xcomp exclusion is required.

Reply:
Fixed.